general/setup/tutorial.md

```diff
@@ -40,4 +40,6 @@ If you have a Mac you'll have a version of Ruby installed already. A coach can h
 You can [download and run the RubyInstaller program](http://rubyinstaller.org/) to program Ruby on Windows.
 
 
+## Python (optional)
 
+Python would usually set up on your operating system. Your tutor will show you how to run Python code.
```

Comment:
`Python usually comes with your operating system. Your coach will show you how to use it to run execute Python code.`